readme.md

```diff
@@ -4,7 +4,7 @@
 
 | Parameter    | Value                                                                    |
 | ------------ | ------------------------------------------------------------------------ |
-| Chain ID     | elystestnet-1                                                            |
+| Chain ID     | elysicstestnet-1                                                         |
```

Comment:
Since we will be having two nets now, do we need to keep this here? Rather we should redirect to networks repo

Reply:
link to the networks repo